Services/UI/OJS.Services.Ui.Data/Implementations/ContestCategoriesDataService.cs

```diff
@@ -20,7 +20,7 @@ public IQueryable<ContestCategory> GetAllVisible()
             => this.GetQuery(cc => cc.IsVisible);
 
         public Task<IEnumerable<T>> GetAllVisible<T>()
-            => this.GetQuery(cc => cc.IsVisible)
+            => this.GetQuery(cc => cc.IsVisible && !cc.IsDeleted)
```

Comment:
`IsDeleted` should be handled by the global query filter.

Reply:
Removed